packages/framework/client-logger/external-telemetry/package.json

```diff
@@ -0,0 +1,75 @@
+{
+	"name": "@fluidframework/external-telemetry",
+	"version": "2.0.0-rc.2.0.0",
+	"description": "External Fluid telemetry types and classes for both creating consuming said telemetry",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/framework/client-logger/external-telemetry"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"main": "dist/index.js",
+	"module": "lib/index.mjs",
+	"types": "dist/index.d.ts",
+	"scripts": {
```

Comment:
Let's capture a follow-up task to add linting to the package, using the strict linting config.